src/Themes/NexusMods.Themes.NexusFluentDark/Styles/UserControls/Spine/SpineStyles.axaml

```diff
@@ -0,0 +1,57 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:spine="clr-namespace:NexusMods.App.UI.Controls.Spine;assembly=NexusMods.App.UI"
+        xmlns:panels="clr-namespace:Avalonia.Labs.Panels;assembly=Avalonia.Labs.Panels">
+    <Design.PreviewWith>
+        <Border Height="300">
+            <Border.Resources>
+                <spine:SpineDesignViewModel x:Key="DesignDataContext" />
+            </Border.Resources>
+
+            <spine:Spine DataContext="{StaticResource DesignDataContext}" />
+        </Border>
+    </Design.PreviewWith>
+
+    <!-- Add Styles Here -->
+    <Style Selector="spine|Spine">
+        <Setter Property="Width" Value="72" />
+        <Setter Property="Margin" Value="0,12,0,12" />
+
+        <Style Selector="^ panels|FlexPanel#SpineFlexPanel">
+            <Setter Property="Direction" Value="Column" />
+            <Setter Property="RowSpacing" Value="12" />
+            
+            <Style Selector="^ > Border#HomeBorder">
+                <Setter Property="panels:Flex.Shrink" Value="0" />
+            </Style>
+            
+            <Style Selector="^ > Border#DownloadBorder">
+                <Setter Property="panels:Flex.Shrink" Value="0" />
+            </Style>
+            
+            <Style Selector="^ > Border#AddBorder">
+                <Setter Property="panels:Flex.Shrink" Value="0" />
+            </Style>
+        </Style>
+
+        <Style Selector="^ Border#LoadoutSelectorBorder">
```

Comment:
Could you add a comment saying that this is for the region that hosts the loadout icons of the spine?

The name `Border` in Avalonia (and WPF for that matter) can sometimes be a misnomer, it's usually used to add an outline to a region of space like we are, however; the name `Border` itself implies it's an outline. Someone from outside of the core team could get confused, thinking that the actual border outline should be growing, when it rather instead is simply that the whole area should be shrinking.